### PR TITLE
Implement auth mechanism for FHIR IG Importer

### PR DESCRIPTION
**Pull request by drizzentic**
## Summary by CodeRabbit

- **New Features**
  - Added support for OpenHIM API authentication with new environment variables: `OPENHIM_API_USERNAME` and `OPENHIM_API_PASSWORD`.

- **Documentation**
  - Updated configuration details to reflect the new authentication parameters for improved system integration with the OpenHIM API.

### CODE REVIEW

**Comment by rcrichton** [comment]
Task linked: [CU-86bzercjw Implement auth mechanism for FHIR IG Importer](https://app.clickup.com/t/86bzercjw)

**Review by brett-onions — APPROVED** [review]
LGMT

File: fhir-ig-importer/package-metadata.json [hunk 1]
```diff
@@ -15,6 +15,8 @@
     "FHIR_IG_IMPORTER_CORE_HOST": "0.0.0.0",
     "FHIR_IG_IMPORTER_CORE_URL": "http://0.0.0.0:3001/fhir/ig/v1.0",
     "FHIR_IG_IMPORTER_UI_VERSION": "latest",
-    "FHIR_IG_IMPORTER_CORE_VERSION": "latest"
+    "FHIR_IG_IMPORTER_CORE_VERSION": "latest",
+    "OPENHIM_API_USERNAME": "<your-openhim-username>",
+    "OPENHIM_API_PASSWORD": "<your-openhim-password>"
```
Comment (bradsawadye) [inline fhir-ig-importer/package-metadata.json:20]: think we should use the openhim default credentials for platform here

**Review by bradsawadye — APPROVED** [review]
looks good just one minor comment

**Pull request merged**